lib/directive/ng_show_hide.dart

```diff
@@ -8,7 +8,7 @@ part of angular.directive;
     selector: '[ng-hide]',
     map: const {'ng-hide': '=>hide'})
 class NgHide {
-  static String NG_HIDE_CLASS = 'ng-hide';
+  static const NG_HIDE_CLASS = 'ng-hide';
```

Comment:
it's superfluous.  but if you like it, feel free to add it back in.  angular doesn't seem consistent about whether it prefers to include these type annotations.